data-model/src/main/java/org/hypertrace/core/datamodel/shared/DataFlowMetrics.java

```diff
@@ -0,0 +1,23 @@
+package org.hypertrace.core.datamodel.shared;
+
+/**
+ * Set of metrics that can be used to track the flow of data from and its processing times as it passes through various
+ * components.
+ */
+public enum DataFlowMetrics {
```

Comment:
I think `Dataflow` should be a single word.
Also, please keep the enum name Singular, `DataflowMetric`

Reply:
made above changes. Intellij is does not like the Dataflow as single word though :)